spec/checkPullRequestReviewSpec.js

```diff
@@ -0,0 +1,1180 @@
+      });
+
+      it('should check type of review', () => {
+        expect(
+          pullRequestReviewModule.handlePullRequestReview
+        ).toHaveBeenCalled();
+      });
+
+      it('should wait for 3 minutes before performing any action', () => {
+        expect(utilityModule.sleep).toHaveBeenCalled();
+        expect(utilityModule.sleep).toHaveBeenCalledWith(THREE_MINUTES);
+      });
+
+      it('should unassign reviewer', async () => {
+        expect(github.issues.removeAssignees).toHaveBeenCalled();
+        expect(github.issues.removeAssignees).toHaveBeenCalledWith({
+          owner: payloadData.payload.repository.owner.login,
+          repo: payloadData.payload.repository.name,
+          issue_number: payloadData.payload.pull_request.number,
+          assignees: [payloadData.payload.review.user.login],
+        });
+      });
+
+      it('should check if all reviewers have approved the PR', () => {
+        expect(github.search.issuesAndPullRequests).toHaveBeenCalled();
+        expect(github.search.issuesAndPullRequests).toHaveBeenCalledWith({
+          owner: payloadData.payload.repository.owner.login,
+          repo: payloadData.payload.repository.name,
+          q: `repo:oppia/oppia review:approved ${payloadData.payload.pull_request.number}`,
+        });
+      });
+
+      it('should assign remaining reviewer', () => {
+        expect(github.issues.addAssignees).toHaveBeenCalled();
+        expect(github.issues.addAssignees).toHaveBeenCalledWith({
+          owner: payloadData.payload.repository.owner.login,
+          repo: payloadData.payload.repository.name,
+          issue_number: payloadData.payload.pull_request.number,
+          assignees: ['reviewer3'],
+        });
+      });
+
+      it('should ping remaining reviewers', () => {
+        expect(github.issues.createComment).toHaveBeenCalled();
+        expect(github.issues.createComment).toHaveBeenCalledWith({
+          owner: payloadData.payload.repository.owner.login,
+          repo: payloadData.payload.repository.name,
+          issue_number: payloadData.payload.pull_request.number,
+          body: 'Assigning @reviewer3 for code owner reviews' + ', Thanks!',
+        });
+      });
+
+      it('should not assign pr author', () => {
+        expect(github.issues.addAssignees).not.toHaveBeenCalledWith({
+          owner: payloadData.payload.repository.owner.login,
+          repo: payloadData.payload.repository.name,
+          issue_number: payloadData.payload.pull_request.number,
+          assignees: [payloadData.payload.pull_request.user.login],
+        });
+      });
+
+      afterAll(() => {
+        payloadData.payload.pull_request.requested_reviewers = initialReviewers;
+        payloadData.payload.pull_request.assignees = initialAssignees;
+      });
+    });
+
+    describe('when all reviewers have approved and none are assigned.', () => {
+      // Note that when the reviewer is the last reviewer, the list of
```

Comment:
Can you add an example - comment isn't that clear.

Reply:
I'm not sure how the example to add since this can only be noticed from the payload. I've modified the comment though.